project_stock/tests/test_project_stock.py

```diff
@@ -76,7 +93,16 @@ def test_project_task_without_analytic_account(self):
         self.task.action_done()
         self.assertFalse(self.task.stock_analytic_line_ids)
 
-    def test_project_task_analytic_lines_without_tags(self):
+    @users("admin")
```

Comment:
No need of this decorator if you keep the other method without renaming nor users decorator (it already run with admin), and only add the other method.